dockerfiles/incr.sh

```diff
@@ -53,7 +53,7 @@ if [[ -n "$FORCE" ]] ; then
   docker image build -t hpccsystems/platform-build:${BUILD_LABEL} --build-arg PREV_LABEL=${HEAD}-Debug --build-arg BASE_VER=7.8 --build-arg BUILD_TYPE=Debug platform-build/
 else
   PREV_COMMIT=$(echo "${PREV}" | sed -e "s/-Debug//")
-  git diff --binary ${PREV_COMMIT} > platform-build-incremental/hpcc.gitpatch
+  git diff --binary ${PREV_COMMIT} ':!./' > platform-build-incremental/hpcc.gitpatch
```

Comment:
@richardkchapman - not 100% sure about this. This is excluding dockerfiles from the diff, since changes there should not require a rebuild to the platform-build-incremental images...